datarobot_batch_scoring/batch_scoring.py

```diff
@@ -265,7 +264,11 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
 
         exit_code = None
 
-        response_handlers = (unpack_data, format_data)
+        try:
+            response_handlers = get_response_handlers_from_url(base_url)
+        except ValueError as e:
+            ui.fatal(str(e))
```

Comment:
I'm only looking at the diff, so forgive me if I missed something.

It looks like no matter what you specify for `--api_version`, it's just going to try to figure it out from the base_url - which doesn't seem like what we want.

Reply:
We don't infer the api version from the user's input, even though the code might be making such impression. Here's what happens:

`batch_scoring --host=http://dr.com --api_version=api/v1` results in `base_url` being `http://dr.com/api/v1` which is then passed to the main `run_batch_predictions` function. 

I could get the `response_handlers` earlier where arguments are parsed, but that'd mean we need to pass `response_handlers` down to `run_batch_predictions`. Adding another parameter to that function is something I wanted to avoid doing, given how unwieldy it already is. Instead I made it dynamically determine the `response_handlers` by looking at the `base_url` parameter it got.

So, instead of 
```python
# dumb function needs to get everything from parameters
run_batch_predictions(base_url, response_handlers) 
```
I made it

```python
# smart function! knows how to handle different api versions in base_url parameter
run_batch_predictions(base_url) 
```

A bit of extra work in exchange for not touching interface of `run_batch_predictions`. Let me know if this doesn't sound like a good reason to you.